syntax = "proto3";

package tc_serialize;

message RouterSettings{
	double bus_wait_time = 1;
	double bus_velocity = 2;
}

message TransportRouter{
	RouterSettings rout_set =3;
}